pages/_app.js

```diff
@@ -39,13 +40,15 @@ export default function App({ Component, pageProps }) {
   return (
     <>
       <GlobalStyle />
-      <Component
-        onAddActivity={handleAddActivity}
-        onEditActivity={handleEditActivity}
-        onDeleteActivity={handleDeleteActivity}
-        activities={activities}
-        {...pageProps}
-      />
+      <Layout>
```

Comment:
I'm really happy to see the inclusion of a `Layout` component.

Reply:
Wanted to write the same :)